framework/src/util/AnyMap.cpp

```diff
@@ -1197,6 +1197,49 @@ namespace cppmicroservices
         return os;
     }
 
+    template <>
+    std::ostream&
+    any_value_to_cpp(std::ostream& os, AnyMap const& m, uint8_t const increment, int32_t const indent)
+    {
+        auto const mapType = m.GetType();
+        std::string typeStr;
+        switch (mapType)
+        {
+            case cppmicroservices::any_map::map_type::ORDERED_MAP:
```

Comment:
Should the "typeStr" values include the fully qualified name? I.e. "cppmicroservices::any_map::map_type::ORDERED_MAP" etc.? Or would I have "using" statements in the surrounding generated code to simplify this output?

Similarly, should the types "AnyMap" and "AnyVector" be fully qualified?

Reply:
They certainly can be fully qualified if you'd like, however I figured for the sake of being concise, I figured we would use Using statements